include/libsupl.rst

```diff
@@ -0,0 +1,153 @@
+.. _libsupl:
+
+LibSUPL
+#######
+
+LibSUPL is an external SUPL (Secure User Plane Location) client library, which is used to retrieve `A-GPS`_ data from a SUPL server using the SUPL protocol.
+
+.. |lib_note| replace:: Therefore, you must download and install it separately.
+
+.. include:: supl_os_client.rst
+   :start-after: supl_details_start
+   :end-before: supl_details_end
+
+See :ref:`download_supl` for information on downloading the external SUPL client library.
+
+Overview
+********
+SUPL is an Enabler, which enables the transfer of assistance and network based positioning data of a SET (SUPL Enabled Terminal) over an IP User Plane bearer.
```

Comment:
```suggestion
SUPL enables the transfer of assistance and network based positioning data of a SUPL Enabled Terminal (SET) over an IP User Plane bearer.
```

Reply:
I'm trying to understand this - what would a SET be in our case? Would that be the DK or Thingy? Then we should add that information:

```suggestion
SUPL enables the transfer of assistance and network based positioning data of a SUPL Enabled Terminal (SET), such as an nRF9160 DK or a Thingy:91, over an IP User Plane bearer.
```

Reply:
@evenl Is the SET the DK or Thingy?